src/lib.rs

```diff
@@ -8,6 +8,11 @@
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
 
+#[macro_use]
```

Comment:
I wonder if we want to work out how to make "dummy logging" for crates that we use in Mentat but also want to publish for third-party consumption, like (e.g. `edn`).  Maybe we can use a feature flag to control "no logging"  and replace the `info!` macros with dummy macros in that case?  I'd be curious to know what the Rust community patterns for that stuff are.